### ISSUE TITLE
[Bug] Server Crash upon entering the end (mod coflict with 'Oh The Biomes You'll Go')

### ISSUE THREAD

**Issue by Fazzoc** [quiqueck/BetterEnd#173]
### What happened?

When using the server outside of the end i.e. overworld, other dimensions, it works perfectly fine, no issues, but when going into the end, the server suddenly crashes. Idk if its a mod conflict or anything like that.

I tried to get help from people on the admincraft subreddit but the advice didn't work.(https://www.reddit.com/r/admincraft/comments/11if1tl/comment/jay1e68/?context=3)

Update: Discovered that there is conflicts with 'Oh The Biomes You'll Go' (older versions didn't have those issues), if thats a bug on BetterEnd's part, i'd be nice for it to be resolved. (I'll also report the but over tho them)

Here are the logs (I hope that's enough info to provide)
Server Log: https://pastebin.com/tdAzbQWw
Server Crash Log: https://pastebin.com/C903tGqX

### BetterEnd

2.1.4

### BCLib

2.1.6

### Fabric API

0.75.1

### Fabric Loader

0.14.14

### Minecraft

1.19.2

### Other Mods

```shell
(Due to a relatively big size of mods, i'll provide screenshots of mods included)
https://imgur.com/a/2525Nkc
```

**Comment by Fazzoc** [comment]
Can this be closed?
I realised that with a bit more research, that's it's tied to a very different issue based on BYG having issues with BetterX Gen, i'll report to the bclib github, BYG

**Comment by GhostIsBeHere** [comment]
Reported by newtrogicNexus at AOCAWOL/BYG/issues/1116

I was also able to fix this issue by disabling betterend:chorus_forest in the configs

**Comment by quiqueck** [comment]
Sorry for getting back late, but this issue should be fixed in the latest releases of BetterEnd (2.1.5 and up).
